test/utils/unit-test-helpers.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2019 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* istanbul ignore file */
+
+const missingParamsError = 1;
+const missingParamsSuccess = 0;
+
+module.exports.missingParamsError = missingParamsError;
+module.exports.missingParamsSuccess = missingParamsSuccess;
+
+module.exports.checkUrlAndMethod = function(options, url, method) {
+  expect(options.url).toEqual(url);
+  expect(options.method).toEqual(method);
+};
+
+module.exports.checkMediaHeaders = function(createRequestMock, accept, contentType) {
+  const headers = createRequestMock.mock.calls[0][0].defaultOptions.headers;
+  expect(headers.Accept).toEqual(accept);
+  expect(headers['Content-Type']).toEqual(contentType);
+};
+
+module.exports.checkUserHeader = function(createRequestMock, baseName, param) {
+  const headers = createRequestMock.mock.calls[0][0].defaultOptions.headers;
+  expect(headers[baseName]).toEqual(param);
+};
+
+module.exports.checkDefaultSuccessArgs = function(createRequestMock) {
+  // get arg to getMissingParams
+  const userParams = createRequestMock.mock.calls[0];
+  expect(typeof userParams[0]).toEqual('object');
+};
+
+module.exports.checkForEmptyObject = function(missingParamsMock) {
```

Comment:
This method needs a more descriptive name.  Check what for empty object?  Maybe "checkUserParamsIsEmptyObject".